Number_types/include/CGAL/MP_Float_impl.h

```diff
@@ -70,6 +70,9 @@ template < typename T >
 inline
 void MP_Float::construct_from_builtin_fp_type(T d)
 {
+    // Fix "...::exp is used uninitialized" errors with Crosstool v16.
```

Comment:
The data member `exp` gets initalized in the constructors for example [here](https://github.com/mhsaar/cgal/blob/c3b42a43c3f62433c3623ca899a562b73130bd60/Number_types/include/CGAL/MP_Float_impl.h#L126).  So why should we get a warning?

Reply:
I agree with @afabri, the function is private and all the calls are done the 3 constructors and all of them initialize exp to 0.

Reply:
Ok, let me check, I created these automatically from the patches I have. It maybe a old fix that is not required anymore.